source/matrix_free/matrix_free.inst.in

```diff
@@ -16,104 +16,352 @@
+        print_memory_consumption<std::ostream>(std::ostream &) const;
+
+    template void MatrixFree<deal_II_dimension,  float, VectorizedArray<float,  1>>::
       print_memory_consumption<ConditionalOStream>(ConditionalOStream &) const;
 
-    template void
-    MatrixFree<deal_II_dimension,
-               float>::print_memory_consumption<std::ostream>(std::ostream &)
-      const;
 
-    template void MatrixFree<deal_II_dimension, float>::
+
+
+    template void MatrixFree<deal_II_dimension, double, VectorizedArray<double, 1>>::internal_reinit<double>(
+              const Mapping<deal_II_dimension> &,
+              const std::vector<const DoFHandler<deal_II_dimension> *> &,
+              const std::vector<const AffineConstraints<double> *> &,
+              const std::vector<IndexSet> &,
+              const std::vector<hp::QCollection<1>> &,
+              const AdditionalData &);
+
+    template void MatrixFree<deal_II_dimension, double, VectorizedArray<double, 1>>::internal_reinit<double>(
+              const Mapping<deal_II_dimension> &,
+              const std::vector<const hp::DoFHandler<deal_II_dimension> *> &,
+              const std::vector<const AffineConstraints<double> *> &,
+              const std::vector<IndexSet> &,
+              const std::vector<hp::QCollection<1>> &,
+              const AdditionalData &);
+
+    template void MatrixFree<deal_II_dimension, float, VectorizedArray<float, 1>>::internal_reinit<double>(
+      const Mapping<deal_II_dimension> &,
+      const std::vector<const DoFHandler<deal_II_dimension> *> &,
+      const std::vector<const AffineConstraints<double> *> &,
+      const std::vector<IndexSet> &,
+      const std::vector<hp::QCollection<1>> &,
+      const AdditionalData &);
+
+    template void MatrixFree<deal_II_dimension, float, VectorizedArray<float, 1>>::internal_reinit<double>(
+      const Mapping<deal_II_dimension> &,
+      const std::vector<const hp::DoFHandler<deal_II_dimension> *> &,
+      const std::vector<const AffineConstraints<double> *> &,
+      const std::vector<IndexSet> &,
+      const std::vector<hp::QCollection<1>> &,
+      const AdditionalData &);
+
+    template void MatrixFree<deal_II_dimension, float, VectorizedArray<float, 1>>::internal_reinit<float>(
+      const Mapping<deal_II_dimension> &,
+      const std::vector<const DoFHandler<deal_II_dimension> *> &,
+      const std::vector<const AffineConstraints<float> *> &,
+      const std::vector<IndexSet> &,
+      const std::vector<hp::QCollection<1>> &,
+      const AdditionalData &);
+
+    template void MatrixFree<deal_II_dimension, float, VectorizedArray<float, 1>>::internal_reinit<float>(
+      const Mapping<deal_II_dimension> &,
+      const std::vector<const hp::DoFHandler<deal_II_dimension> *> &,
+      const std::vector<const AffineConstraints<float> *> &,
+      const std::vector<IndexSet> &,
+      const std::vector<hp::QCollection<1>> &,
+      const AdditionalData &);
+
+
+
+
+
+
```

Comment:
Remove all but 1 (or 3 if you insist in separation between the blocks) empty lines.